.github/workflows/main-branch-notifications.yml

```
@@ -0,0 +1,24 @@
+name: Main Branch Notifications
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)